### DISCUSSION TITLE
Internal links not working for me (sometimes)

### DISCUSSION THREAD

**Discussion by heyteresagreen** [ObsidianPublisher/obsidian-github-publisher#240] [Q&A — Plugin]
I really like this plugin, but I'm not sure if I just can't find the correct link settings or if there is a bug or conflict somewhere. 

My current settings are:

- Use Wikilinks: ON
- Internals links: OFF
- Wikilinks to MDlinks: OFF

This is working correctly on [one page](https://teresawatts.com/now) but not on [another](https://teresawatts.com/letters-7) where the Wikilinks aren't rendering. With a different combination of settings, I get all internal links in the form "Post name.md" instead of the slug version, even with that setting turned on. I am using the Digital Garden Jekyll theme and some of my notes are in folders, but I could have sworn it was working before.

Any idea what could be up with it?

I'm using the most recent version of the plugin and the Obsidian app.

**Comment by Mara-Li** [comment]
You need to use a theme that support wikilinks. I'm not support other theme than mine. Please, open an issue on the digital garden theme.
